openmdao/docs/openmdao_book/features/debugging/profiling/index.ipynb

```diff
@@ -62,7 +62,32 @@
     "    # do some other stuff that I don't want to profile/trace...\n",
     "```\n",
     "\n",
-    "* [Timing Systems under MPI](timing.ipynb)\n"
+    "* [Timing Systems under MPI](timing.ipynb)\n",
+    "\n",
+    "## Dumping debug info to file based on MPI rank and/or PID\n",
+    "\n",
+    "The `OPENMDAO_DUMP` environment variable can be set to a number of values to control what information\n",
+    "is dumped and which file it is written to. `OPENMDAO_DUMP` can have values like `stdout`, `stderr`, \n",
```

Comment:
this should probably say the value ca be a comma separated list of keywords (as is implied by the examples )

Reply:
fixed